src/averageAges.js

```diff
@@ -36,8 +52,17 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const filteredWomenPeople = people.filter(el => {
```

Comment:
Do not abbreviate names to unreadable ones.
```suggestion
  const filteredWomen = people.filter(person => {
```